sim/xcelium/__init__.py

```diff
@@ -79,12 +82,21 @@ def get_verilog_models(self) -> List[str]:
     return verilog_sim_files
         
   def fill_outputs(self) -> bool:
+    saif_opts = self.extract_saif_opts()
+    wav_opts = self.extract_waveform_opts()[1]
+
     self.output_waveforms = []
     self.output_saifs = []
     self.output_top_module = self.top_module
     self.output_tb_name = self.get_setting(f"{self.sim_input_prefix}.tb_name")
     self.output_tb_dut = self.get_setting(f"{self.sim_input_prefix}.tb_dut")
     self.output_level = self.get_setting(f"{self.sim_input_prefix}.level")
+    
+    if saif_opts ["mode"] is not None:
+      self.output_saifs.append(os.path.join(self.run_dir, "ucli.saif"))
+    if wav_opts["type"] is not None:
+      self.output_waveforms.append(os.path.join(self.run_dir, f'{wav_opts["dump_name"]}.vcd'))
```

Comment:
This extension depends on the waveform type?

Reply:
Fixed!